### PR TITLE
Updated Program to output results for BP Tables

### PR DESCRIPTION
**Pull request by jdev42092**
Added code to:
1) Output ACS 2011-2015 population counts/ denominators for population
shares
2) Output # of rental units from ACS
3) Separated number of single family and condo units from RealProp
4) Added code to output 1990, 2000, and 2010 Census figures on total
population and household count from NCDB

### CODE REVIEW

**Comment by jdev42092** [comment]
@NeighborhoodInfoDC/review This program (which outputs results for our Bridge Park Data Tables) is ready for review.

**Comment by ptatian** [comment]
Will review

File: Prog/BPKPresentationResults.sas [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( ACS )
+%DCData_lib( Realprop )
+%DCData_lib( Police )
+%DCData_lib( NCDB )
+
+libname Bridgepk "L:\Libraries\Bridgepk\Data\";
```
Comment (ptatian) [inline Prog/BPKPresentationResults.sas]: You shouldn't need to use a libname statement here. You should use %DCData_lib() as with the other libraries.

File: Prog/BPKPresentationResults.sas [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+run;
+
+
+** Pull Property and Violent Crime Stats for City **;
+data Crime_prop_stats_city (keep= city property_crime_rate_2000 property_crime_rate_2001
+property_crime_rate_2002 property_crime_rate_2003 property_crime_rate_2004 
+property_crime_rate_2005 property_crime_rate_2006 property_crime_rate_2007 
+property_crime_rate_2008 property_crime_rate_2009 property_crime_rate_2010
+property_crime_rate_2011 property_crime_rate_2012 property_crime_rate_2013
+property_crime_rate_2014 property_crime_rate_2015);
```
Comment (ptatian) [inline Prog/BPKPresentationResults.sas]: Should be able to add 2016 crime data.

**Review by ptatian — CHANGES_REQUESTED** [review]
Looks good, Jay. Just a few small changes. Thanks.

**Comment by jdev42092** [comment]
@ptatian Thanks for those suggestions, Peter. I incorporated both of them. Is this ok to merge and batch submit?

**Comment by ptatian** [comment]
Yes, please go ahead and merge. Thanks.

**Pull request merged**